components/tooltip/tooltip.js

```diff
@@ -729,8 +728,9 @@ class Tooltip extends RtlMixin(LitElement) {
 		if (this.for) {
 			const targetSelector = `#${cssEscape(this.for)}`;
 			target = ownerRoot.querySelector(targetSelector);
-			target = target || (ownerRoot && ownerRoot.host && ownerRoot.host.querySelector(targetSelector));
+			target = target || ownerRoot?.host?.querySelector(targetSelector);
 		} else {
+			console.warn('<d2l-tooltip>: missing required attribute "for"');
```

Comment:
This seems like a good thing to me, but willing to revert this too if there's disagreement.

Reply:
I don't feel strongly one way or another. It's nice to avoid the inconsistency, but on the other hand, it's also nice not to force consumers to create a non-interactive wrapper element just for the `for`.

Reply:
Ok, I'm going to merge this. If it proves more annoying than helpful we can remove it and indicate `for` is required _if_ the target isn't the parent/host.